src/views/Lottery/index.tsx

```diff
@@ -42,15 +42,15 @@ const Lottery = () => {
 
   return (
     <LotteryPage>
-      <PageSection background={TITLE_BG} svgFill={theme.colors.overlay} index={1} hasCurvedDivider={false}>
+      <PageSection background={TITLE_BG} index={1} hasCurvedDivider={false}>
         <Hero />
       </PageSection>
       <PageSection
         containerProps={{ style: { marginTop: '-30px' } }}
         background={GET_TICKETS_BG}
         concaveDivider
-        concaveBackgroundLight="#7645D9"
-        curvePosition="top"
+        clipFill={{ light: '#7645D9' }}
```

Comment:
If I am reading this correctly `clipFill` is supposed to be a `string`.

Reply:
Looking at `PageSection/types.ts`

Reply:
I think it's all good hachi? It's an object with two keys - light & dark, both of which are strings? 

```
{
  light: string
  dark?: string
}
```

You might have spotted something I've missed though!

Reply:
Ah yeah you're right. My bad.